zeapp/android/src/main/java/de/berlindroid/zeapp/zeui/zehome/ZePages.kt

```diff
@@ -73,59 +73,86 @@ internal fun ZePages(
             TemplateChooserDialog(vm, templateChooser, modifier = Modifier.padding(paddingValues))
         }
 
-        LazyColumn(
-            state = lazyListState,
-            modifier =
-                Modifier
-                    .padding(top = paddingValues.calculateTopPadding()),
-            contentPadding =
-                PaddingValues(
-                    start = paddingValues.calculateStartPadding(LayoutDirection.Ltr),
-                    end = paddingValues.calculateEndPadding(LayoutDirection.Ltr),
-                    bottom = paddingValues.calculateBottomPadding(),
-                ),
-        ) {
-            if (message.isNotEmpty()) {
-                stickyHeader {
-                    InfoBar(message, messageProgress, vm::copyInfoToClipboard)
-                }
+        ZePagesLazyList(
+            modifier = Modifier.padding(top = paddingValues.calculateTopPadding()),
+            paddingValues = paddingValues,
+            lazyListState = lazyListState,
+            slots = uiState.slots,
+            sendPageToBadgeAndDisplay = vm::sendPageToBadgeAndDisplay,
+            slotToBitmap = vm::slotToBitmap,
+            customizeSlot = vm::customizeSlot,
+            customizeSponsorSlot = vm::customizeSponsorSlot,
+            resetSlot = vm::resetSlot,
+        )
+
+        if (message.isNotEmpty()) {
+            InfoBar(
+                modifier = Modifier.align(Alignment.BottomCenter),
```

Comment:
To change the InfoBar position, just change the Alignment on this line. e.g. if you want to put it on the top the aligment would be `Aligment.TopCenter`